packages/components/src/components/NewModal/NewModal.tsx

```diff
@@ -30,7 +30,7 @@ const NEW_MODAL_CONTENT_ID = 'modal-content';
 const MODAL_ELEVATION = 0;
 const ICON_SIZE = 40;
 
-const Container = styled.div<{ $elevation: Elevation; $size: NewModalSize }>`
+const Container = styled.div<{ $elevation: Elevation; $size: NewModalSize; $customWidth?: number }>`
```

Comment:
We don't want to allow custom width.

Reply:
It's understandable, but the modal width in the mockup is 445px. So, tiny(400px) is too small and breaks the title. Do you suggest to use small(600px) instead?

Reply:
removed https://github.com/trezor/trezor-suite/pull/14410/commits/bd37af68e33b5bc592b302ac122ae7a831e65962